webCommander/centralHub/views.py

```diff
@@ -35,12 +38,28 @@ def list_twitter(request):
 def handle_summary(request, twitter_handle):
     user = TwitterUser.objects.get(twitter_handle=twitter_handle)
 
-    #TODO Get posts from database and render them
     posts = user.user_posts.all()
-    breakpoint
+
     #TODO If no posts get last 20 posts from user
+    if len(posts) == 0:
+        last_tweets = get_all_tweets_from_user(user.twitter_handle, max_results=20)
```

Comment:
I would put the 20 in a constant

Reply:
naming: `get_all_tweets_from_user` implies all, but you only get 20 so I would rename this to `get_tweets_from_user`